### PR TITLE
feat: added PersonsAuthResetJob, added interface Job

### PR DESCRIPTION
**Pull request by PavelVesnin**

### CODE REVIEW

File: __schema__/index.graphql [hunk 1]
```diff
@@ -525,9 +545,10 @@ type Mutation {
   mergeLegalEntities(input: MergeLegalEntitiesInput!): MergeLegalEntitiesPayload
 
   "Resets an authentication method for a single `Person` using its globally unique ID."
-  resetPersonAuthenticationMethod(
-    input: ResetPersonAuthenticationMethodInput!
-  ): ResetPersonAuthenticationMethodPayload
+  resetPersonAuth(input: ResetPersonAuthInput!): ResetPersonAuthPayload
```
Comment (ephor) [inline __schema__/index.graphql]: It will be a breaking change

File: __schema__/persons.graphql [hunk 2]
```diff
@@ -98,19 +98,19 @@ type PersonEdge {
 }
 
 """
-Input for `resetPersonAuthenticationMethod` mutation.
+Input for `resetPersonAuthInput` mutation.
 
 In order to reset patient authentication method user must have a scope `person:reset_authentication_method`
 """
-input ResetPersonAuthenticationMethodInput {
+input ResetPersonAuthInput {
```
Comment (ephor) [inline __schema__/persons.graphql]: This is also will be a breaking change

File: __schema__/index.graphql (hunk 1 above)
Comment (kiurchv) [reply __schema__/index.graphql]: Maybe we should not rename this mutation?

File: __schema__/personsAuthResetJobs.graphql [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+An object for PersonsAuthResetJob.
+"""
+type PersonsAuthResetJob implements Node & Job {
+  "The ID of an object"
+  id: ID!
+  "Primary key identifier from the database"
+  databaseId: UUID!
+  "Job name."
+  name: String
+  "Legal Entity Merge Job status."
```
Comment (ephor) [inline __schema__/personsAuthResetJobs.graphql]:
```suggestion
  "Persons Reset Authentification Job status."
```

**Pull request merged**